src/main/java/com/google/security/cryptauth/lib/securegcm/Ed25519.java

```diff
@@ -208,20 +207,20 @@ public static void validateAffinePoint(BigInteger[] p) throws Ed25519Exception {
     };
   }
 
-  /**
-   * Converts a point in affine representation to extended representation
-   */
-  @VisibleForTesting
+  /** Converts a point in affine representation to extended representation */
+  // TODO(b/120887495): This @VisibleForTesting annotation was being ignored by prod code.
```

Comment:
please search-replace references to internal bug numbers

Reply:
if this is an OSS-related concern, it should not be, because cross tool says it is ok to have b/ references on OSS code. It helps tracking issues. If there is a different project-specific reason, I could scrub that.